src/main/java/hudson/plugins/active_directory/ActiveDirectorySecurityRealm.java

```diff
@@ -734,7 +735,7 @@ private LdapContext bind(String principalName, String password, SocketInfo serve
         public DirContext createDNSLookupContext() throws NamingException {
             Hashtable env = new Hashtable();
             env.put(Context.INITIAL_CONTEXT_FACTORY, "com.sun.jndi.dns.DnsContextFactory");
-            env.put("java.naming.provider.url", "dns:");
+            env.put(Context.PROVIDER_URL, "dns:");
```

Comment:
Unrelated code cleanup done while I was here: using a built-in constant where possible, consistent with the previous usage near line 666.